docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.7"
```

Comment:
Why is this docker-compose file included?

Reply:
To easily adapt the keycloak realm config for testing when necessary,